lib/commands/util/git-credentials-helper/index.js

```diff
@@ -27,7 +27,13 @@ class GitCredentialsHelperCommand extends AbstractCommand {
         return ['profile', 'debug'];
     }
 
-    handle(cmd, cb) {
+    handle(cmd, cb, remaining) {
+        if (remaining && remaining[0] !== 'get') {
+            const NON_GET_OPERATION_ERR = `The ask-cli git credentials helper only supports "get". Operation "${remaining[0]}" is not supported.`;
+            Messenger.getInstance().error(NON_GET_OPERATION_ERR);
+            return cb(NON_GET_OPERATION_ERR);
+        }
+
```

Comment:
the git operations are like the below, this condition will still make this error show

```
38f9d3582200:cdT2 chihying$ GIT_TRACE_SETUP=true GIT_TRACE=true git push

~

15:19:57.872677 run-command.c:663 trace: run_command: 'ask util git-credentials-helper --profile default get'

15:20:00.482634 run-command.c:663 trace: run_command: 'ask util git-credentials-helper --profile default store'

[Error]: The ask-cli git credentials helper only supports "get". Operation "store" is not supported.

~

After the code pushed, please check the deployment status
via Alexa Developer console:
https://developer.amazon.com/alexa/console/ask/build/custom/amzn1.ask.skill.9ba5945b-8774-42d0-a276-6524bfe1d091/development/en_US
Everything up-to-date
```

Same with `git pull`

Reply:
Based on the doc (https://git-scm.com/docs/gitcredentials#Documentation/gitcredentials.txt-codegetcode) ,
we can return error if not `get` and  `store`, 
For the `store` logic, I have ever checked with AHS team, even though they put credentials in ask_config in v1, but they set expired time as zero, so the store operation doesn't affect at all. That's the reason, I didn't handle `store` option before.